fenwick_tree/cpp/fenwick_tree.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <vector>
+using namespace std;
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpt3szxfti/fenwick_tree/cpp/fenwick_tree.cpp
+++ b/tmp/tmpt3szxfti/fenwick_tree/cpp/fenwick_tree.cpp
@@ -5,7 +5,7 @@
 struct FenwickTree {
 	vector<int> v;
 	int length;
-	
+
 	FenwickTree(int l) : length(l) {
 		v.assign(length + 1, 0);
 	}
```